modules/virt-accessing-vmi-yaml-ssh.adoc

```diff
@@ -0,0 +1,140 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-consoles.adoc
+
+[id="virt-accessing-vmi-yaml-ssh_{context}"]
+= Accessing a virtual machine instance via SSH with YAML configurations
+
+You can enable a SSH connection to a VMI without the need to run the `virtctl expose` command. The YAML file for the VMI should include a label with a value that is identical to the value for the `selector` in the service YAML file. Then, the service forwards the SSH traffic connections to the VMI.
```

Comment:
I think you can drop "connections", and just say: "...forwards the SSH traffic to the VMI."

Reply:
Removed "connections".